packages/victory-native/src/components/victory-voronoi-container.tsx

```diff
@@ -46,13 +46,15 @@ VictoryVoronoiContainer.defaultEvents = (
         ? {}
         : handler(event, { ...props, ...targetProps }, eventKey, context);
 
+  const voronioHelper = new VoronoiHelpers();
```

Comment:
I'm having a similar issue and need this fix too and came across this PR. Thanks for working on this!

One thing I noticed: doesn't this change break throttling though? We're now creating a new instance of the throttling function each time `defaultEvents` is called when `VoronoiHelpers` is instantiated.